virttest/qemu_vm.py

```diff
@@ -881,6 +881,10 @@ def set_dev_params(dev, dev_params,
                 dev.set_param("id", dev_id)
 
             dev_type = "virtio-rng-pci"
+            machine_type = self.params.get("machine_type", "")
```

Comment:
"pc" is better default value for machine_type

Reply:
OK.